WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/UsageDetails/CouponRestrictionsViewModel.swift

```diff
@@ -9,6 +9,8 @@ final class CouponRestrictionsViewModel: ObservableObject {
 
     let currencySymbol: String
 
+    let shouldDisplayLimitUsageToXItemsRow: Bool
```

Comment:
I think this property should be a computed variable based on `discountType`. This will be important since I'm adding the option to update the discount type in #6935 - so this property will no longer be constant. Using a computed variable makes sure that `shouldDisplayLimitUsageToXItemsRow` will always be updated when the discount type is updated.

```suggestion
    var shouldDisplayLimitUsageToXItemsRow: Bool {
        discountType != .fixedCart
    }
```